chia/full_node/generator.py

```diff
@@ -62,9 +62,9 @@ def setup_generator_args(self: BlockGenerator) -> Tuple[SerializedProgram, Progr
     return self.program, args
 
 
-def run_generator(self: BlockGenerator, max_cost: int) -> Tuple[int, SerializedProgram]:
```

Comment:
the "unsafe" counterpart to this function (below) is specifically not renamed in this patch. This is because changing `run_generator()` -> `run_generator_mempool()` in the same patch as also renaming `run_generator_unsafe()` -> `run_generator()` runs the risk of calls to the first function changing to calls to the second one, without anything showing up in the patch.

I do think we *should* rename it to either just `run_generator()` or `run_generator_non_mempool()` or something like that. But not in this patch.